### DISCUSSION TITLE
improve better AOE recognition of BM hunter using pets.

### DISCUSSION THREAD

**Discussion by Dragonfo** [Hekili/hekili#1405] [Ideas]
Hello, now at the moment the addon recognizes 50/50 AOE from BM pets. I would like to suggest adding an ability like Stomp to tracking. When we use Multi-Shot, we activate Stomp on pets. And I would like that the addon was interacted with the recognition of nearby targets through Stomp.

**Comment by Hekili** [comment]
Are you not using pet-based target detection?

**Reply by Dragonfo** [reply]
I only use pet-based target detection. It's just that when my pets move literally 1-2m away from the target, the addon no longer understands whether there is AoE or not, but as soon as I just switch to the next target next to the boss, the addon starts showing AoE and correctly counting targets. That's why I suggested adding another Stomp to the tracking. So to speak as a help.
